clickfile.py

```diff
@@ -507,7 +507,7 @@ def run(name, jobs, numprocesses, ui_item, amount, users, network):
         if numprocesses:
             command = f"{command} --numprocesses {numprocesses} --dist loadgroup"
     elif name == "tracer":
-        command = "py.test integration/tests/tracer"
+        command = "py.test -n 50 integration/tests/tracer"
```

Comment:
50 is too much, we don't have a servers with 50 cpus.
Please setup -n auto and xdist will run thread by cpus